### DISCUSSION TITLE
Error install with laravel 6.x

### DISCUSSION THREAD

**Discussion by caocanhlinh** [spatie/crawler#367] [Q&A]
Hi,
I am currently having problems with the installation for version 5 and 6.

v6
```
 Problem 1
    - spatie/crawler[6.0.0, ..., v6.x-dev] require illuminate/collections ^8.17 -> satisfiable by illuminate/collections[v8.17.0, ..., 8.x-dev].
    - illuminate/collections[v8.17.0, ..., 8.x-dev] require illuminate/contracts ^8.0 -> found illuminate/contracts[v8.0.0, ..., 8.x-dev] but these were not loaded, likely because it conflicts with another require.
    - Root composer.json requires spatie/crawler ^6.0 -> satisfiable by spatie/crawler[6.0.0, 6.0.1, 6.0.2, v6.x-dev].
```

v5
```
Problem 1
    - illuminate/collections[v8.6.0, ..., 8.x-dev] require illuminate/contracts ^8.0 -> found illuminate/contracts[v8.0.0, ..., 8.x-dev] but these were not loaded, likely because it conflicts with another require.
    - spatie/crawler 5.0.0 requires illuminate/collections ^8.6 -> satisfiable by illuminate/collections[v8.6.0, ..., 8.x-dev].
    - Root composer.json requires spatie/crawler v5.0 -> satisfiable by spatie/crawler[5.0.0].
```

help me! Thanks

**Comment by captain-burah** [comment]
Need more info. There are certain versions that may not work. I suggest you install Laravel 6.20.
`composer create-project --prefer-dist laravel/laravel projectName "6.2"`

If it still persist then install the package and update composer,
`composer require spatie/laravel-sitemap:4.0`
`composer update`

**Reply by caocanhlinh** [reply]
Thanks, now i have installed version 4.7.6 .  I don't know if it is possible to upgrade to the next version?

**Reply by captain-burah** [reply]
Glad to help! 
Do note the versions of laravel before 5.6 are considered outdated. There may be many deprecated libraries in it. Also make sure your using the latest php version.
